paper/paper.md

```diff
@@ -127,12 +129,14 @@ In pyGeo, these are referred to as _local_ design variables because a single con
 Conversely, it is also common to define geometric operations involving a collection of control points across the entire FFD block.
 These are referred to as _global_ design variables in pyGeo.
 For example, twist variables can be defined as rotations of the control points about a reference axis that runs along the wing. <!-- MM: this is a good example but I feel we need to specify what twist is to a non-aerospace audience-->
-
 \autoref{fig:FFD_DV} shows a few common planform design variables for an aircraft wing.
-Parameterizations based on the singular value decomposition are also possible [@Wu2022b].
+
+Design variables formulated from groupings of FFD control points often exhibit ill conditioning. 
+To alleviate this, a parameterization based on singular value decomposition is also possible [@Wu2022b].
 <!-- talk about ref axis more? -->
+<!-- [ ] TODO AY-: I suggest moving the sentence on planform changes to be merged with the previous paragraph, and moving the SVD statement elsewhere. It does not flow well -->
```

Comment:
@anilyil I agree that the SVD sentence is awkward. Since it's a pretty standalone feature, it doesn't really fit anywhere besides the basic FFD section and I think the only way to introduce it is to set up the motivation for it, which relates to the DV section. Is this a good way to do that?